feg/gateway/services/s6a_proxy/servicers/test/test_s6a_server.go

```diff
@@ -109,6 +109,10 @@ func testHandleAIR(settings *sm.Settings) diam.HandlerFunc {
 		if err != nil {
 			fmt.Printf("AIR Unmarshal for message: %s failed: %s", m, err)
 			code = diam.UnableToComply
+		} else if expectedVisitedPLMNID != req.VisitedPLMNID {
+			// Flags needs to exist for this test
+			fmt.Printf("error: VisitedPLMNID (%v) doesnt match with the expected VisitedPLMNID (%v)\n", req.VisitedPLMNID, expectedVisitedPLMNID)
+			code = diam.UnableToComply
```

Comment:
added this to the test server to make sure received PLMN is what we sent.
Before we were just sending 0 and not checking